tdrs-backend/tdpservice/settings/common.py

```diff
@@ -271,6 +276,14 @@ class Common(Configuration):
     es_logger = logging.getLogger('elasticsearch')
     es_logger.setLevel(getattr(logging, LOGGING_LEVEL))
 
+    PARSER_LOGGER = logging.getLogger('tdpservice.parsers')
+
+    AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
+    AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
+    AWS_REGION = os.getenv("AWS_REGION", "us-gov-west-1")
+    AWS_S3_BUCKET_NAME = os.getenv("AWS_S3_BUCKET_NAME")
+    AWS_S3_LOGS_PREFIX = os.getenv("AWS_S3_LOGS_PREFIX", "logs")
```

Comment:
a lot of these are already defined in `settings/local.py` when using `USE_LOCALSTACK=yes`, can we reuse those definitions?